govuk-one-login/tech-docs
# AP-362--Fix-incorrect-content-on-the-Register-and-manage-your-service-page

[AP-362](https://govukverify.atlassian.net/browse/AP-362)

## Why

Correct errors in the published guidance at https://docs.sign-in.service.gov.uk/before-integrating/register-and-manage-your-service/

## What

- remove references to the registration page, this is no longer used in integration
- link directly to the self service tool to register and create a client
- add points 2 and 3 to cover confirmation of the email address and mobile number that are not covered by the table
- move all external and internal links into partials/_links.erb
- remove redundant paragraph duplicating content about the ID token signing algorithm
- remove redundant update your services details section that was using the old form, all changes should be via the self service tool
- 
## Technical writer support

Do you need a tech writer's support, for example to review your PR?

## How to review

Tell reviewers how to assess your changes.

- build, check links and run locally using nginx by running `./preview-with-nginx.sh`
- ensure page http://localhost/before-integrating/register-and-manage-your-service/ makes sense

## Changelog

no changelog entry required for fix

## Confirm



[AP-362]: https://govukverify.atlassian.net/browse/AP-362?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
